news/_posts/releases/2020-12-03-prestashop-1-7-7-0-available.md

```diff
@@ -0,0 +1,213 @@
+- [Product quantity no updated if a discount with free gift is applied](https://github.com/PrestaShop/PrestaShop/issues/21506)
+- [Recalculate shipping costs option enabled - the Shipping price not instantly updated](https://github.com/PrestaShop/PrestaShop/issues/21267)
+- [Payment block not refreshed after adding product](https://github.com/PrestaShop/PrestaShop/issues/21293)
+- [Incorrect quantity added to product when it is added to the same order as a gift also](https://github.com/PrestaShop/PrestaShop/issues/21531)
+- [Incorrect Total's product line when Round type = "Round on each item"](https://github.com/PrestaShop/PrestaShop/issues/21708)
+- [When we try to return products in all shops context, an exception is displayed](https://github.com/PrestaShop/PrestaShop/issues/21990)
+- [An alert "Error! There are not enough products in stock." is displayed when updating the product quantity in "all shops" context](https://github.com/PrestaShop/PrestaShop/issues/21987)
+- [The characters limit is wrongly calculated when choosing a predefined message](https://github.com/PrestaShop/PrestaShop/issues/22086)
+- [In the info block - The shipping is not refreshed after adding a product](https://github.com/PrestaShop/PrestaShop/issues/22071)
+- [While creating an order, after adding a product, if you click quickly, the focus change between the "up" arrow and the "down" arrow](https://github.com/PrestaShop/PrestaShop/issues/22026)
+- [With a different currency than the default one, if there is a  discount on a category for a customer group, when we update a product the price is wrong](https://github.com/PrestaShop/PrestaShop/issues/22096)
+- [Details modal for carts - the Refresh button is not working](https://github.com/PrestaShop/PrestaShop/issues/22105)
+- [An unexpected error occurred when adding to the cart a product with customization in a multistore context](https://github.com/PrestaShop/PrestaShop/issues/22110)
+- [The shipping block is not instantly updated when updating the Address](https://github.com/PrestaShop/PrestaShop/issues/22124)
+
+**Front-office**
+- [Pack of products - Incorrect image displayed for products with combinations](https://github.com/PrestaShop/PrestaShop/issues/21875)
+- [displayAfterProductThumbs hook content is duplicated after adding a product in the cart](https://github.com/PrestaShop/PrestaShop/issues/22113)
+
+**Back-office**
+- [Adding incorrect language causes exception in Edit Currency page](https://github.com/PrestaShop/PrestaShop/issues/21891)
+- [Bad URL for module catalog page after disabling MBO](https://github.com/PrestaShop/PrestaShop/issues/21588)
+
+
+## Acknowledgments
+
+PrestaShop is above all a community project: from the 119 committers who contributed to this release, 57 are new contributors. The vast majority of PrestaShop contributors are not directly affiliated with the PrestaShop company.
+
+@formarion
```

Comment:
@marionf Sorry I couldn't build the page on my env so I couldn't see how it would look, and I don't know how you want to display these so I just copied the data here of:
- all contributors
- new contributors

This needs to be edited for better display 😉

Reply:
Thank you, I removed the new contributors part as they are already in all contributors and it is repetitive to put them back, especially since the list is already long